test/cipher_test.exs

```diff
@@ -1,7 +1,7 @@
 require Cipher.Helpers, as: H  # the cool way
 
 defmodule CipherTest do
-  use ExUnit.Case, async: true
+  use ExUnit.Case
```

Comment:
I removed the async: true because the test for runtime config is modifying the global config values so there's a chance of causing tests to fail if running concurrently.